src/Files.App/Data/Models/AppModel.cs

```diff
@@ -127,10 +129,9 @@ public string PCloudDrivePath
 
 		/// <summary>
 		/// Gets or sets a value indicating the AppWindow DPI.
-		/// TODO update value if the DPI changes
 		/// </summary>
-		private float _AppWindowDPI = Win32PInvoke.GetDpiForWindow(MainWindow.Instance.WindowHandle) / 96f;
-		public float AppWindowDPI
+		private float _AppWindowDPI = PInvoke.GetDpiForWindow((HWND)MainWindow.Instance.WindowHandle) / 96f;
+		public float AppWindowDPI // TODO: Update value if the DPI changes
```

Comment:
Regarding the TODO, I don't believe this isn't such a common occurrence that we need to update when the value changes.

Reply:
I can remove it if you like.

Reply:
Thanks!